kolibri/plugins/facility/assets/src/views/DataPage/index.vue

```diff
@@ -22,6 +22,8 @@
               class="download-button"
               @click="downloadSessionLog"
             />
+            <span v-if="noSessionLogs">{{ $tr('noLogsYet') }}</span>
+            <GeneratedElapsedTime v-else :date="sessionDateCreated" />
```

Comment:
See if this change works to prevent the "generated 50 years ago" bug

```suggestion
            <GeneratedElapsedTime v-else-if="sessionDateCreated" :date="sessionDateCreated" />
```